servers/dsub/jobs/test/base_test_cases.py

```diff
@@ -438,3 +454,146 @@ def test_query_jobs_submission_pagination(self):
                     page_token=response.next_page_token,
                     extensions=ExtendedQueryFields(submission=min_time)),
                 [job2])
+
+        def assert_status_counts_equal(self, counts1, counts2):
+            counts1Dict = {}
+            for status_count in counts1.counts:
+                counts1Dict[status_count.status] = status_count.count
+
+            counts2Dict = {}
+            for status_count in counts2.counts:
```

Comment:
nit: should be able to replace these with dictionary comprehensions

`{sc.status: sc.count for sc in counts2.counts}`